tutorial/docker-compose-postgres.yaml

```diff
@@ -10,10 +10,15 @@ services:
     image: quay.io/debezium/kafka:${DEBEZIUM_VERSION}
     ports:
      - 9092:9092
```

Comment:
Since 9092 is only advertised in the Docker network, and not reachable from the host, this should be removed

Reply:
done. thanks